sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/SingleFileReportImporter.java

```diff
@@ -39,24 +38,24 @@ protected SingleFileReportImporter(String reportPathKey, String msg) {
   }
 
   @Override
-  public final void importReport(SensorContext context, Map<String, Integer> numberOfLinesOfCode) {
+  public final void importReport(SensorContext context) {
     String reportPath = context.settings().getString(reportPathKey);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)